config/log4j-file.properties

```diff
@@ -6,6 +6,7 @@ log4j.rootLogger=WARN, default.file
 log4j.appender.default.file=io.confluent.ksql.util.TimestampLogFileAppender
 log4j.appender.default.file.ImmediateFlush=true
 log4j.appender.default.file.append=false
-log4j.appender.default.file.file=${ksql.log.dir}/cli-%timestamp.log
+
+log4j.appender.default.file.file=${ksql.log.dir}/cli/cli-%timestamp.log
```

Comment:
Should the directory be called `ksql` instead of `cli`?

Reply:
It's ksql-cli. I changed it to `ksql-cli`.

Reply:
That would be my other suggestion. Cool, thx!